server/sonar-server/src/main/java/org/sonar/server/computation/design/DsmDataBuilder.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * SonarQube, open source software quality management tool.
+ * Copyright (C) 2008-2014 SonarSource
+ * mailto:contact AT sonarsource DOT com
+ *
+ * SonarQube is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * SonarQube is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.server.computation.design;
+
+import org.sonar.graph.Dsm;
+import org.sonar.graph.DsmCell;
+import org.sonar.server.computation.step.ComponentUuidsCache;
+import org.sonar.server.design.db.DsmDb;
+
+public final class DsmDataBuilder {
+
+  private final Dsm<Integer> dsm;
+  private final ComponentUuidsCache uuidByRef;
+
+  private final DsmDb.Data.Builder dsmBuilder;
```

Comment:
Variable should be renamed. It's misleading with DsmDataBuilder. I suggest dbBuilder or something like that. It should show that it relates to DB.